cli/cmd/cluster_create.go

```diff
@@ -82,20 +80,18 @@ func createCluster(kotsRestClient *kotsclient.VendorV3Client, opts kotsclient.Cr
 
 	if ve != nil && len(ve.Errors) > 0 {
 		if len(ve.SupportedDistributions) > 0 {
-			return nil, fmt.Errorf("%s\n\nSupported Kubernetes distributions and versions are:\n%s", errors.New(strings.Join(ve.Errors, ",")), supportedDistributions(ve.SupportedDistributions))
-		} else {
-			return nil, fmt.Errorf("%s", errors.New(strings.Join(ve.Errors, ",")))
+			print.ClusterVersions("table", r.werr, ve.SupportedDistributions)
```

Comment:
Why is this printing to stderr?

Reply:
Because the original implementation was also sending it to stderr.
@marccampbell ok if we change this to stdout for showing the supported ones. And only do the actual error to stderr?

Reply:
I don't see where this line prints an error.  None of this reads like an error, except the output name

```
print.ClusterVersions("table", r.werr, ve.SupportedDistributions)
```

Reply:
That looks like supported distros should to go to STDOUT and the error returned below should go to STDERR.

Reply:
Fixed